main.tf

```diff
@@ -234,6 +234,13 @@ resource "google_pubsub_subscription" "push_subscriptions" {
         audience              = lookup(each.value, "audience", "")
       }
     }
+
```

Comment:
This is the main change - adding the no_wrapper and write meta block https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/pubsub_subscription.html#write_metadata